silkworm/node/snapshots/path.hpp

```diff
@@ -32,7 +34,7 @@ namespace silkworm::snapshots {
 constexpr int kFileNameBlockScaleFactor{1'000};
 
 //! The segment size measured as number of blocks included in each segment
-constexpr uint64_t kDefaultSegmentSize{500'000};
+constexpr std::array kDefaultSegmentSizes{500'000u, 100'000u};
```

Comment:
Namespace constants in headers should be `inline`. See https://abseil.io/tips/140